vcf2maf.py

```diff
@@ -357,10 +357,13 @@ def resolve_vcf_allele_depth_values(mapped_sample_format_data, vcf_alleles, vari
 				read_depth_value = ""
 			allele_depth_values[i] = read_depth_value
 
-	# 4. Strelka (INDEL): handle VCF INDEL lines by Strelka, where variant allele depth is in TIR
+	# 4. Strelka (INDEL): handle VCF INDEL lines by Strelka, where variant allele depth is in TIR, reference allele depth is in TAR.
 	elif is_strelka_indel_vcf(vcf_format_data_keys):
-		# reference allele depth is not provided by Strelka for indels, so we have to skip it
+		# pick tier1 counts if a comma-delimited value
 		allele_depth_values[variant_allele_idx] = mapped_sample_format_data["TIR"].split(",")[0]
+		if variant_allele_idx == 1: ref_allele_idx = 0
+		else: ref_allele_idx = 1
```

Comment:
```suggestion
		if variant_allele_idx == 1:
		    ref_allele_idx = 0
		else:
		    ref_allele_idx = 1
```
i haven't seen much single line if/else in python

As a side note: I also noticed that the formatting in this file uses tabs, for Python spaces is more common. It might make sense introduce a code formatter at some point (see e.g. https://black.readthedocs.io)

Reply:
Thanks @inodb. This is great! Formatted the code with Black.